README.md

```diff
@@ -90,7 +94,7 @@ jobs:
           fetch-depth: 0                        # Includes all history for all branches and tags
 
       - id: get-version
-        uses: im-open/git-version-lite@v2.1.3
+        uses: im-open/git-version-lite@v2.2.0
```

Comment:
This was mentioned before, but your comments will need to include one of the +semver keywords to make this bump happen.

Reply:
Actually I take that back, it looks like it did increment it properly from some of the comments.